pkgs/tools/X11/smart-wallpaper/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, xdpyinfo
+, killall
+, xwinwrap
+}:
+
+stdenv.mkDerivation {
+  pname = "smart-wallpaper";
+  version = "unstable-2022-09-01";
+
+  src = fetchFromGitHub {
+    owner = "Baitinq";
+    repo = "smart-wallpaper";
+    rev = "d175695d3485fb14144c1908eb3569b20caa6ba5";
+    sha256 = "sha256-cFgvuntdKPzdQJ7xE2DIT+aNAazocIhM6BBbcQOlryU=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -Dm755 -t $out/bin smart-wallpaper
```

```suggestion
    install -Dm755 -t $out/bin smart-wallpaper
```
-D already does that